packages/react-scripts/utils/getCSSModuleLocalIdentWithProjectName.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const crypto = require('crypto');
+
+const getCSSModuleLocalIdent = require('react-dev-utils/getCSSModuleLocalIdent');
+
+const createHash = buffer =>
+  new Buffer(crypto.createHash('md5').update(buffer).digest('hex'))
+    .toString('base64')
+    .substr(0, 5);
+
+module.exports = projectName => (...args) => {
+  const localIdent = getCSSModuleLocalIdent(...args);
+
+  // Create an ident based on the file location, class name and Project Name.
+  // Will be unique across projects
+  // Format [ file/folder ]_[ localName ]__[ 5 character hash of CRA ident and project name ]
+  return (
+    localIdent.substring(0, localIdent.length - 5) +
```

Comment:
Is there a worry here that if it increases from 5 or decreases this could cause problems?

Reply:
ha, me and @mbfisher had that discussion. It's a possibility.

Reply:
I guess we can just keep an eye on how their function changes and the work they are doing. I am confident they wouldn't change it too, but you never know